cmd/daemon/main.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
should we add some file header info like below? I see it in many files but this is not followed in all files. Since this is new file, should we consider it?
``
/*
Copyright 2021.
Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at
    http://www.apache.org/licenses/LICENSE-2.0
.....
*/

Reply:
Added